### PR TITLE
add change avatar function

### PR DESCRIPTION
**Pull request by tgsmdww**

### CODE REVIEW

File: src/pages/UserCenter/Edit/LocalAvatarButton.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    padding: 5px;
+  }
+`
+
+export default ({ handleAvatarSubmit }: Props) => {
+  const [open, setOpen] = useState(false)
+  const [imgSrc, setImgSrc] = useState('')
+  const fileInputRef = useRef<HTMLInputElement>(null)
+
+  let fileType = ''
```
Comment (mwindson) [inline src/pages/UserCenter/Edit/LocalAvatarButton.tsx]: Is it better to make `fileType` as a component state?

File: package.json [hunk 2]
```diff
@@ -20,13 +20,15 @@
     "@material-ui/icons": "^3.0.1",
     "@reach/router": "^1.2.1",
     "@sentry/browser": "^4.4.2",
+    "@types/react-cropper": "^0.10.3",
```
Comment (Deturium) [inline package.json]: @tgsmdww `@types` 是开发依赖

File: src/pages/UserCenter/Edit/LocalAvatarButton.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import React, { useState, useRef } from 'react'
+import styled from 'styled-components'
+
+import { Button, Dialog } from '@material-ui/core'
+
+import LocalAvatarButton from './LocalAvatarBox'
```
[the remark this answers is not in the document]
Comment (Dearkano) [reply src/pages/UserCenter/Edit/LocalAvatarButton.tsx]: 然后整个项目crash了。。

[the remark this answers is not in the document]
Comment (Deturium) [reply src/pages/UserCenter/Edit/LocalAvatarButton.tsx]: XD 规范层面

**Pull request merged**